api/src/main/java/io/lumeer/engine/api/constraint/NumberConstraintType.java

```diff
@@ -51,88 +54,130 @@
     * Number format respecting given locale.
     */
    private NumberFormat numberFormat = NumberFormat.getNumberInstance();
-   private NumberFormat integerNumberFormat = NumberFormat.getNumberInstance();
+   private NumberFormat bigNumberFormat = NumberFormat.getNumberInstance();
+   private NumberFormat integerNumberFormat = NumberFormat.getIntegerInstance();
+   private NumberFormat currencyNumberFormat = NumberFormat.getNumberInstance();
 
    public NumberConstraintType() {
-      integerNumberFormat.setParseIntegerOnly(true);
+      ((DecimalFormat) currencyNumberFormat).setParseBigDecimal(true);
+      ((DecimalFormat) bigNumberFormat).setParseBigDecimal(true);
```

Comment:
For now there is none, in the future there might be. I am not sure if we want to add currency or so. So I kept that separated.